packages/ui/src/common/components/Pagination/Pagination.tsx

```diff
@@ -12,9 +13,15 @@ interface PaginationProps {
 }
 
 export const Pagination: FC<PaginationProps> = ({ pageCount = 0, handlePageChange, page }) => {
+  const history = useHistory()
+
   if (pageCount < 2) {
     return null
   }
+  const pageChangeAction = (value: any) => {
+    history.push(`?page=${value.selected + 1}`)
+    handlePageChange(value.selected + 1)
```

Comment:
Maybe i'm missing something: Why can page only increase?

Reply:
value.selected is zero index based so for page 1 value.secleted is zero